src/instructlab/utils.py

```diff
@@ -44,6 +44,37 @@ class TaxonomyReadingException(Exception):
     """An exception raised during reading of the taxonomy."""
 
 
+class Message(TypedDict):
+    """
+    Represents a message within an AI conversation.
+    """
+
+    content: str
```

Comment:
Where is content populated? Am I missing something?

Reply:
It's one of the fields that gets populated within the dataset generated by SDG.

Reply:
For context, the way `TypedDicts` work is you just declare a class with the fields it should have and then you place a hint on a dict that you go to use. See this example from [their documentation](https://mypy.readthedocs.io/en/stable/typed_dict.html):

```py
from typing import TypedDict

Movie = TypedDict('Movie', {'name': str, 'year': int})

movie: Movie = {'name': 'Blade Runner', 'year': 1982}
```